### PR TITLE
New command: `bootstrap-init`

### PR DESCRIPTION
**Pull request by dwhswenson**
This provides a CLI interface for the `FullBootstrap` initialization from OpenPathSampling core, which is used in the MSTIS/MISTIS OPS example notebooks.

### CODE REVIEW

**Comment by dwhswenson** [comment]
This is ready for review and comment. Without review, I will merge this after at least 24 hours, no sooner than Wed 28 Aug 12:00 GMT (14:00 my local).

File: paths_cli/parameters.py [hunk 1]
```diff
@@ -109,6 +109,15 @@ def get(self, storage, names):
...
 
+MULTI_INIT_SNAP = OPSStorageLoadMultiple(
+    param=Option('-f', '--init-frame',
+                 help="identifier for potential initial snapshots"),
+    store='snapshots',
+    value_strategies=[GetByName('tags'), GetByNumber('snapshots')],
+    none_strategies=[GetOnlySnapshot(),
+                     GetPredefinedName('tags', 'initial_snapshot')]
+)
+
```
Comment (sroet) [inline paths_cli/parameters.py:120]: Is this actually used somewhere?

**Review by sroet — APPROVED** [review]
one question about new parameter, feel free to ignore, LGTM otherwise

File: paths_cli/parameters.py (hunk 1 above)
Comment (dwhswenson) [reply paths_cli/parameters.py]: No; the original idea in this branch was to write something that would be a single command to set up an entire network (which could require multiple initial snapshots). But that's a little too messy, and so I'd rather have the user set up one transition at a time.

Removed `MULTI_INIT_SNAP` in  8352adc.

**Pull request merged**